arm/Microsoft.Sql/servers/.parameters/parameters.json

```
@@ -18,6 +18,15 @@
                 "secretName": "administratorLoginPassword"
             }
         },
+        "administrators": {
+            "value": {
+                "azureADOnlyAuthentication": false,
+                "login": "John Doe",
+                "sid": "<<objectId>>",
```

To work in the test environment we ideally set it to the deploying service principal.

```suggestion
                "sid": "<<deploymentSpId>>",
```